package.json

```diff
@@ -168,9 +169,9 @@
     "@commitlint/cli": "^18.0.0",
     "@commitlint/config-conventional": "^18.0.0",
     "@compodoc/compodoc": "^1.1.19",
-    "@design-factory/design-factory": "^15.2.2",
+    "@design-factory/design-factory": "^17.0.0",
     "@ng-bootstrap/ng-bootstrap": "^16.0.0",
-    "@ng-select/ng-select": "~12.0.0",
+    "@ng-select/ng-select": "~12.0.7",
```

Comment:
should not be needed anymore

Reply:
it's still needed as for ` @agnos-ui/style-bootstrap` to prevent the build to break with unresolved imports
I've moved them in ` .yarnrc.yml`